agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java

```diff
@@ -75,6 +76,14 @@ private static synchronized void installExportersFromJar(final String exporterJa
               .readEnvironmentVariables()
               .readSystemProperties()
               .build();
+      TraceConfig activeTraceConfig = OpenTelemetrySdk.getTracerProvider().getActiveTraceConfig();
```

Comment:
I am not sure this is the right place for this code. This method is called `installExportersFromJar`, does not seem to be related to sampling. I would create a new method, e.g. `configure` or `configureSampler`, and call it from `installAgentTracer` method. Independently of exporter.

Reply:
good thought, makes sense 👍

Reply:
Done

Reply:
Currently `configure` method is called from `installExportersFromJar`, which is only called when `exporterJar` is configured. Does not seem right to me. Call `configure` from `installAgentTracer`?

Reply:
Sorry. Totally missed it. Running whole tests suite in local takes more time. I am running `googleJavaFormat`, `integration:test` and `integration:latestDepTest`. Is there any selected set of tests that can be run before pushing?

Reply:
Not really :( I don't think we have a set of "important" tests to run when changing such base functionality. Any ideas are very welcomed!

Reply:
Ah Yes. Agreed that changes in base functionality requires the full suite to be run and I didn't know what exactly needed to be run before raising a PR. I was more looking for a kind of checklist to be verified before raising a PR. Do you think that is useful or Is it just me?

Reply:
I think it would be mildly useful :) E.g. I don't ever run `latestDepTest` locally before making PR. `googleJavaFormat` is certainly needed and then just test your changes with corresponding `test` task. Not sure if there are any more required steps.